.travis.yml

```diff
@@ -1,24 +1,38 @@
 # Linting/Testing available at https://config.travis-ci.com/explore
 os: linux
 dist: xenial
+
+stages:
+  - Linting
+  - test
+  # This assumes tags will only be used for version releases
+  - name: "deploy-github"
+    if: "tag IS present"
+  - name: "deploy-pypi"
+    if: "tag IS present"
+
 services:
   - postgresql
   - redis
+
+# The test stage will be a matrix of the python list * the env['jobs'] list
+language: "python"
+python:
+  - "3.6"
+  - "3.7"
+  - "3.8"
+  - "3.9"
+
+env:
+  global:
+    - INVOKE_NAUTOBOT_LOCAL=True
+  jobs:
+    - "TESTING_OPTIONS='--failfast --buffer'"
+    - "TESTING_OPTIONS='--failfast --buffer --label nautobot.core.tests.integration'"
```

Comment:
Presumably for this one we also need to set `NAUTOBOT_INTEGRATION_TEST=1`? Travis run shows a single (skipped) test case since this isn't set here.